src/DepOutputChannel.ts

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+import * as vscode from 'vscode';
+
+export class DepOutputChannel {
+  outputChannel: vscode.OutputChannel;
+  constructor(channelName = 'Dependency Analytics') {
+    if (!this.outputChannel) {
+      this.outputChannel = vscode.window.createOutputChannel(channelName);
+    }
+  }
+
+  getOutputChannel(): vscode.OutputChannel {
+    this.outputChannel.clear();
+    return this.outputChannel;
+  }
+
+  showOutputChannel() {
+    this.outputChannel.show();
+  }
+
+  clearOutputChannel() {
```

Comment:
why are you clearing the output channel?

Reply:
Am making use of outputChannel to show `stdout/stderr` for each stack analysis so clearing would make sense before each run

Reply:
you can write eg: https://github.com/redhat-developer/vscode-adapters/blob/master/src/extension.ts#L17 for stdout/stderr

Reply:
Currently am writing stdout/stderr in same channel as in https://github.com/fabric8-analytics/fabric8-analytics-vscode-extension/pull/284/files#diff-55eabb84e751500cb765a140d212c608R191 , i don't think we need two channel's here